libs/langchain-openai/src/azure/chat_models.ts

```diff
@@ -1,3 +1,4 @@
+import process from "node:process";
```

Comment:
Oh hmm actually this will break non-node environments - can we use this instead? https://github.com/langchain-ai/langchainjs/blob/main/langchain-core/src/utils/env.ts#L40

Reply:
My bad, I forgot about this!
I sent an update using `getEnv` that fixes that